native/core/src/support.rs

```diff
@@ -10,5 +10,15 @@ pub struct RestoreToken {
     /// Server homebase url
     pub homeurl: Url,
     /// Session to hand to client
-    pub session: Session,
+    pub session: CustomAuthSession,
```

Comment:
eeeer... does this mean the old stored tokens do still deserialize?

Reply:
Yes, I introduced `CustomAuthSession` to support token in older format.